default.nix

```diff
@@ -9,5 +9,6 @@ in
       pkgs.git
       pkgs.shellcheck
       pkgs.stack
+      pkgs.haskellPackages.haskell-language-server
```

Comment:
This looks like a drive-by. You could choose to add this in a separate PR, but in this case I'm fine with having it in this PR. Could you sort it to be between `pkgs.git` and `pkgs.shellcheck`?